src/main/java/uk/gov/mca/beacons/service/dto/WrapperDTO.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.mca.beacons.service.dto;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class WrapperDTO<T> {
+
+  public Map<String, Object> getMeta() {
+    return meta;
+  }
+
+  public ArrayList<T> getData() {
+    return data;
+  }
+
+  private Map<String, Object> meta;
+  private ArrayList<T> data;
+
+  public WrapperDTO() {
+    meta = new HashMap<String, Object>();
+    data = new ArrayList<T>();
+  }
+
+  public void AddMeta(String key, String value) {
```

Comment:
Method names should be kebab-case so: `addMeta` and similarly for the `addData` method below.

Should be able to do this refactor in IntelliJ with shortcut `F6` I believe